cloud/src/meta-service/txn_kv.cpp

```diff
@@ -408,6 +408,7 @@ TxnErrorCode Transaction::get(std::string_view key, std::string* val, bool snaps
     }
 
     if (!found) return TxnErrorCode::TXN_KEY_NOT_FOUND;
+    num_get_keys_++;
```

Comment:
should be added right after `fdb_future_get_value` is called, no mater what it returns, we count it as a `get`